website/src/views/venues/VenuesContainer.tsx

```diff
@@ -46,18 +54,16 @@ export const VenuesContainerComponent: FC<Props> = ({ venues }) => {
   const location = useLocation();
   const matchParams = useParams<Params>();
 
-  // Search state
-  const [
-    /** Value of the controlled search box; updated real-time */
-    searchQuery,
-    setSearchQuery,
-  ] = useState<string>(() => qs.parse(location.search).q || '');
-  /** Actual string to search with; deferred update */
-  const deferredSearchQuery = searchQuery; // TODO: Redundant now. Use React.useDeferredValue after we adopt concurrent mode
+  const [searchQuery, setSearchQuery] = useState<string>(() => qs.parse(location.search).q || '');
```

Comment:
/nit 

```suggestion
  const [searchQuery, setSearchQuery] = useState(() => qs.parse(location.search).q || '');
```

Reply:
That doesn't work because `qs.parse` returns `any`. We could do the below type assertion instead, but the current code seems good enough

```suggestion
  const [searchQuery, setSearchQuery] = useState(
    () => (qs.parse(location.search) as Params).q || '',
  );
```